src/lib/driver/message_handler.ts

```diff
@@ -31,13 +31,11 @@ export class DriverMessageHandler {
       case DriverCommand.updateLogConfig:
         driver.updateLogConfig(message.config);
         clientsController.clients.forEach((cl) => {
-          if (cl !== client) {
-            cl.sendEvent({
-              source: "driver",
-              event: "log config updated",
-              config: dumpLogConfig(driver, cl.schemaVersion),
-            });
-          }
+          cl.sendEvent({
+            source: "driver",
+            event: "log config updated",
+            config: dumpLogConfig(driver, client.schemaVersion),
```

Comment:
Let's still only generate the log config once

Reply:
We are? or maybe I am misunderstanding what you mean?

Reply:
doh, got it! will submit a fix soon